### ISSUE TITLE
Login failed, possibly due to invalid credentials or settings: Authentication Error

### ISSUE THREAD

**Issue by ncc-hyperion** [oSumAtrIX/DownOnSpot#97]
When trying to run the script I'm  getting the following error: "Login failed, possibly due to invalid credentials or settings: Authentication Error"

I've changed nothing and the script has worked flawlessly before.

I've double checked my dev secret ID/Pass and everything is correct

**To Reproduce**

1. Go to terminal, run the command '/Users/xxxx/DownOnSpot/target/release/down_on_spot https://open.spotify.com/album/7aPCbAB9hPwnk45jKfIUjN'
4. See error:
Settings successfully loaded.
Continuing with spotify account: <redacted>
Login failed, possibly due to invalid credentials or settings: Authentication Error

**Expected behavior**
Downloads to start as normal

**Desktop (please complete the following information):**

- OS: Mac OS 14.4.1 (23E224)

**Comment by jernauH** [comment]
Exactly the same issue here on Ubuntu Linux 22.04, compiled and ran the latest 0.5.1 version and same authentication issue. I also tried onthespot and also getting an authentication issue for the same user account. I tried a different user account on both apps and same result, logging in via a browser on the same OS using both of the accounts worked fine. I wonder if the authentication scheme has changed with an additional application check or something similar?
